raiden/network/proxies/token.py

```diff
@@ -10,7 +10,7 @@
 from raiden.network.rpc.transactions import check_transaction_threw
 from raiden.utils import safe_gas_limit
 from raiden.utils.typing import Address, Balance, BlockSpecification, TokenAddress, TokenAmount
-from raiden_contracts.constants import CONTRACT_HUMAN_STANDARD_TOKEN
+from raiden_contracts.constants import CONTRACT_CUSTOM_TOKEN
```

Comment:
I really doubt I can do this here.

Reply:
During tests, I have to mint tokens for deposits in ServiceRegistry.

Shall I write a new proxy custom_token.py for tests?

Reply:
@palango is it important to have `setURL` function tested in this client repo?  Otherwise, I remove the test, and avoid introducing the test custom token here.

Reply:
It will not be called by the client at all, so it makes sense to test it only in the contracts repo.

Reply:
Alright!  That simplifies a lot.

Reply:
However, in order to keep `test_service_registry_random_pfs` and `test_configure_pfs`, I need a `ServiceRegistry` with some services registered on it.

Reply:
So I'm keeping `CUSTOM_TOKEN` here.

Reply:
As discussed in the chat it should not affect the production code since the ABI is a superset of ERC-20.